reverse_engineering/helpers/convertToJsonSchemas.js

```diff
@@ -38,17 +44,23 @@ const convertSchema = ({ schema, namespace = EMPTY_NAMESPACE, defaultValue, cust
 	}
 
 	const type = _.isString(schema) ? schema : schema.type;
-	const attributes =  _.isString(schema) ? {} : schema;
-	const field = convertType(namespace, type, getFieldAttributes({ attributes, type, defaultValue, customProperties }));
+	const attributes =  setDefaultValue(_.isString(schema) ? {} : schema, fieldAttributes?.default);
```

Comment:
moved setting default value to this function from the convertType to prevent code duplication